terraform/layer2-k8s/templates/loki-stack-values.tmpl

```diff
@@ -60,10 +60,10 @@ grafana:
       requiredDuringSchedulingIgnoredDuringExecution:
         nodeSelectorTerms:
         - matchExpressions:
-          - key: node.kubernetes.io/lifecycle
+          - key: eks.amazonaws.com/capacityType
```

Comment:
Let's delete all values related to Grafana at all (except enabled: false), because we install it with kube-prometheus-stack.

Reply:
I think it's better to do it in different PR